gems/sorbet/lib/serialize.rb

```diff
@@ -337,6 +337,20 @@ def from_method(method)
           uniq += 1
         end
       end
+
+      # Sanitize parameter names that are not valid.
+      # Ruby 2.7+ argument forwarding works by expanding `foo(...)`
+      # to `foo(**, ****, &&)`, in other words, to a rest parameter named
+      # `*`, a keyword rest parameter named `**` and a block argument named
+      # `&`. Those are all illegal parameter names which we should NOT
+      # generate. (Coincidentally, that is why Ruby 2.7+ uses those names,
+      # so that user code cannot mess with the forwarded arguments)
+      name = name.to_s.gsub(/[^a-zA-Z0-9_]/) do
+        result = '_' + (uniq == 0 ? '' : uniq.to_s)
+        uniq += 1
+        result
+      end
```

Comment:
Should we instead detect the existence of those names specifically?

I don't want to have to figure out whether this `gsub` is too eagerly removing certain characters (maybe certain unicode characters are valid here idk)

Reply:
Sure, that makes sense. I'll change it to that.

Reply:
Done